src/main/java/nl/tudelft/ewi/devhub/server/database/entities/issues/Issue.java

```diff
@@ -0,0 +1,42 @@
+package nl.tudelft.ewi.devhub.server.database.entities.issues;
+
+import java.net.URI;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import nl.tudelft.ewi.devhub.server.database.entities.User;
+
+
+@Data
+@Entity
+@Table(name="issues")
+@EqualsAndHashCode(callSuper = true)
+@NoArgsConstructor
+public class Issue extends AbstractIssue {
+
+	@NotNull	
+	@Column(name="title", length=128)
```

Comment:
I agree that the title should not be too big. But constraints also force us to check them. Maybe we should just leave the default value of `255` characters here, the storage penalty is not too bad anyway.

Also, I think it was a mistake that PR's cannot have names right now. Therefore I think this field should move to `AbstractIssue`.